ci/verify_examples.sh

```diff
@@ -32,6 +32,7 @@ run_examples () {
     examples=$(find . -mindepth 1 -maxdepth 1 -type d -name "$TESTFILTER" ! -iname "_*" ! -name "$TESTEXCLUDES" | sort)
     for example in $examples; do
         pushd "$example" > /dev/null || return 1
+        chmod go+r -R . || return 1
```

Comment:
yep - this was the other way - this is done for a reason

Reply:
or rather - there is another place where the opposite of this is done - to mimick the env of users with strict umask settings - so we could temp removed that